tests/units/txBatch/bulker.test.ts

```diff
@@ -0,0 +1,383 @@
+import { constants, Wallet } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+
+import addresses from "@morpho-labs/morpho-ethers-contract/lib/addresses";
+
+import { MorphoAaveV3Adapter } from "../../../src";
+import CONTRACT_ADDRESSES from "../../../src/contracts/addresses";
+import { AdapterMock } from "../../../src/mocks";
+import { Underlying } from "../../../src/mocks/markets";
+import BulkerTxHandler from "../../../src/txHandler/Bulker.TxHandler";
+import { Bulker } from "../../../src/txHandler/Bulker.TxHandler.interface";
+import { TransactionType } from "../../../src/types";
+import { ADAPTER_MOCK } from "../../mocks/mock";
+
+describe("bulker", () => {
+  const userAddress = Wallet.createRandom().address;
+  let bulkerHandler: BulkerTxHandler;
+
+  beforeEach(async () => {
+    const adapter = MorphoAaveV3Adapter.fromMock(ADAPTER_MOCK);
+    bulkerHandler = new BulkerTxHandler(adapter);
+    await adapter.connect(userAddress);
+    await adapter.refreshAll();
+    expect(bulkerHandler.getOperations()).toHaveLength(0);
+  });
+  describe("supply", () => {
+    it("should add operations to the batch", async () => {
+      //  handle dai supply
+      bulkerHandler.addOperation({
+        type: TransactionType.supplyCollateral,
+        underlyingAddress: Underlying.dai,
+        amount: parseUnits("100"),
+      });
+      expect(bulkerHandler.getOperations().length).toBnGt(0);
```

Comment:
```suggestion
      expect(bulkerHandler.getOperations().length).toEqual(1);
```

Reply:
I just want to test if there is any action, not if there is the correct number of action